alicloud-cpi.html.md.erb

```diff
@@ -0,0 +1,278 @@
+---
+title: Alicloud CPI
+---
+
+This topic describes cloud properties for different resources created by the Alicloud CPI.
+
+## <a id='azs'></a> AZs
+
+Schema for `cloud_properties` section:
+
+* **availability_zone** [String, required]: Availability zone to use for creating instances. Example: `cn-beijing-a`.
+
+Example:
+
+```yaml
+azs:
+- name: z1
+  cloud_properties:
+    availability_zone: cn-beijing-a
+```
+
+---
+## <a id='networks'></a> Networks
+
+Schema for `cloud_properties` section used by dynamic network or manual network subnet:
+
+* **vswitch_id** [String, required]: VSwitch ID in which the instance will be launched. Example: `vsw-0xipoq1c5q04z7xggxwcn`.
+* **security_group_ids** [String, required]: List [Security Group](https://www.alibabacloud.com/help/zh/doc-detail/25468.htm), by ID, to apply to all VMs placed on this network. Example: `["sg-0xiecbyds24g3b9dn14p"]`
+
+Example of manual network:
+
+```yaml
+networks:
+- name: private
+  type: manual
+  subnets:
+  - range: 172.16.0.0/24
+    gateway: 172.16.0.1
+    ip: 172.168.0.3
+    cloud_properties:
+        vswitch_id: vsw-0xipoq1c5q04z7xggxwcn
+        security_group_ids: ["sg-0xiecbyds24g3b9dn14p"]
+```
+
+Example of dynamic network:
+
+```yaml
+networks:
+- name: default
+  type: dynamic
+  cloud_properties:
+    vswitch_id: vsw-0xipoq1c5q04z7xggxwcn
+    security_group_ids: ["sg-0xiecbyds24g3b9dn14p"]
+```
+
+Example of vip network:
+
+```yaml
+networks:
+- name: default
+  type: vip
+  ip: 47.47.47.47
```

Comment:
`ip` is not specified in this section.

Reply:
Hi @cppforlife thanks for your feedback and I have modify it.